server/server/models/__init__.py

```diff
@@ -1,6 +1,79 @@
 """
 SQLAlchemy models for database
 """
+import zope.sqlalchemy
+import transaction
+from pyramid.threadlocal import get_current_registry
+from sqlalchemy import engine_from_config
+from sqlalchemy.orm import configure_mappers, sessionmaker
 from sqlalchemy.ext.declarative import declarative_base
```

Comment:
One minor comment. It's not in PEP8, but it's common to sort all imports in alphabetical order keeping 2 groups: `import module` and `from module import ...`. In your case it will be:
```
import transaction
import zope.sqlalchemy
from pyramid.threadlocal import get_current_registry
from sqlalchemy import engine_from_config
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy.orm import configure_mappers, sessionmaker
```